packages/demo/__stories__/theming.tsx

```diff
@@ -11,16 +10,19 @@ storiesOf('Theming/Editor UI', module).add('Initial State', () => {
     state: [{ plugin: 'text' }]
   }
 
-  const theme: EditorTheming = {
-    textColor: '#222',
-    backgroundColor: '#d9edf7',
-    buttonBackgroundColor: 'transparent',
-    highlightColor: '#007ec1'
+  const theme: CustomEditorTheme = {
```

Comment:
New type `CustomEditorTheme` that makes nearly everything optionally (because merging)